packages/auth/lib/Sessions.js

```diff
@@ -69,7 +69,7 @@ const sessionByToken = async ({ pgdb, token, email: emailFromQuery }) => {
     `, token)
 
   if (sessions && sessions.length > 0) {
-    if (sessions[0].sess.email !== emailFromQuery) {
+    if (sessions[0].sess.email.toLowerCase() !== emailFromQuery.toLowerCase()) {
```

Comment:
```suggestion
    if (!emailFromQuery || sessions[0].sess.email.toLowerCase() !== emailFromQuery.toLowerCase()) {
```

Reply:
why is this neccesarry when you don't lowercase in [auth/lib/Users.js](https://github.com/orbiting/backends/pull/277/files#diff-0eeae7f2c73b01d9b94ce4e17b94977cR296)?

Reply:
An email in `sessions[0].sess.email` can be a user entered input and is case sensitive. Same for  `emailFromQuery`.

Meanwhile [auth/lib/Users.js](https://github.com/orbiting/backends/pull/277/files#diff-0eeae7f2c73b01d9b94ce4e17b94977cR296) is case insensitive.